site_config/blog.js

```diff
@@ -52,6 +52,13 @@ export default {
         desc: 'This blog introduces how to use annotations',
         link: '/blog/translate-dubbo-annotations.md',
       },
+      {
+        title: 'dubbo-zk.md',
```

Comment:
title should be 'Using Zookeeper in Dubbo'.

Reply:
Got it!